model/stats/db.go

```diff
@@ -786,6 +796,34 @@ func (filter StatsFilter) buildMatchStageForTask() bson.M {
 		DbTestStatsIdProjectKeyFull:   filter.Project,
 		DbTestStatsIdRequesterKeyFull: bson.M{"$in": filter.Requesters},
 	}
+
+	// After cutting over the below build variants to use resmoke spawned
+	// with jasper (SERVER-54315), test names are now random UUIDs with
+	// with a human-readble display name. For the period between
+	// 09/01/21-09/13/21, the historical test stats calculations failed to
+	// account for this (EVG-15396). We should ignore any test stats
+	// affected by this bug.
+	affectedVariants := []string{
+		"enterprise-rhel-80-64-bit",
+		"enterprise-rhel-80-64-bit-dynamic-required",
+	}
+	if len(filter.Tasks) == 0 || len(utility.StringSliceIntersection(filter.Tasks, affectedVariants)) > 0 {
+		match["$or"] = []bson.M{
```

Comment:
if there is pagination will this get overwritten [later](https://github.com/evergreen-ci/evergreen/blob/dd3199cc8f9ee0d3316f8b1251a3d956a3b3ab83/model/stats/db.go#L800)?

Reply:
Good catch, I fixed it.